src/components/alert/alert.js

```diff
@@ -91,6 +92,8 @@ export default {
       let dismissCountDown = parseInt(show, 10) || 1
       this.countDownTimerId = setInterval(() => {
         if (dismissCountDown < 1) {
+          this.$emit('dismiss-count-down', dismissCountDown)
+          this.$emit('input', dismissCountDown)
           this.dismiss()
```

Comment:
Since `this.dismiss()` is called here, it will end up with duplicate `input` and `dismiss` events from line 66 and 67

Reply:
I can remove the `input` event on line 96 - should be fine then?

Reply:
That should help.